core/state.py

```diff
@@ -1497,6 +1497,31 @@ def GetAllVars(self):
           result[name] = str_val.s
     return result
 
+  def GetAllCells(self, lookup_mode = scope_e.Dynamic):
```

Comment:
Also I use the style of `foo='default'` (no space).


 I suppose at some point we should do something about issue #1 (autoformatting for Python).  If enough contributors want it, I will be motivated to :)

Reply:
Oh, thank you! Hm, [PEP 8](https://www.python.org/dev/peps/pep-0008/#other-recommendations) (mentioned in #1) says that spaces are not recommended for keyword arguments and default arguments.

> *PEP-8 - Other Recommendations*
>
> Don't use spaces around the = sign when used to indicate a keyword argument, or when used to indicate a default value for an unannotated function parameter.

 Updated e2c89a00.